ios/PhyWeb/UI/PWBeaconsViewController.m

```diff
@@ -329,6 +329,21 @@ - (void)_updateBeaconsNow {
 - (void)_sort {
   BOOL stableMode = [[PWBeaconManager sharedManager] isStableMode];
 
+  BOOL hasRank = NO;
+  for(PWBeacon * beacon in _beacons) {
+    if ([beacon hasRank]) {
+      hasRank = YES;
```

Comment:
I think that with the latest change on the metadata server, we're probably not seeing the issue any more.
What happened before was append_invalid() added beacons that did not have a rank.
(see https://github.com/google/physical-web/commit/62321566ee833815864bf95bbc63adc65e1f1868#diff-32f4822b465eace8648030b1fec3eb2c)

I'm not sure if we should land it since it's going to be improved on server side very soon.